components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -47,7 +48,7 @@
  Snackbars prefer an application's main window is a subclass of @c MDCOverlayWindow. When a standard
  UIWindow is used an attempt is made to find the top-most view controller in the view hierarchy.
  */
-@interface MDCSnackbarManager : NSObject
+@interface MDCSnackbarManager : NSObject <MDCElevationOverriding>
```

Comment:
Shouldn't we also be conforming to MDCElevatable?

Reply:
i dont want this object to get currentElevation as the category goes up the view hierarchy and not NSObjects.

Reply:
How would we know what the `mdc_currentElevation`  within the theming extension?